application/F3DStarter.cxx

```diff
@@ -361,6 +361,11 @@ void F3DStarter::LoadFile(f3d::loader::LoadFileEnum load)
       // Setup the camera according to options
       this->Internals->SetupCamera(fileAppOptions);
     }
+
+    if (!fileName.empty())
+    {
+      this->Internals->Engine->getWindow().setWindowName(fileName + " - " + F3D::AppName);
```

Comment:
I wonder if we should separate `AppName` with `ExecutableName`. The application name should definitely be `F3D` as we state everywhere, the executable name is `f3d` though. Wdyt @Meakk ?

Reply:
AppName is currently used for two things:

1. the `f3d` string in `f3d --help` and such (should be lower case)
2. the `f3d` string in `f3d --version` and such (should be upper case IMO)

Reply:
>     1. the `f3d` string in `f3d --help` and such (should be lower case)

Is there specific reasons not to just go with `argv[0]` for that usage?

Reply:
Yup it should be uppercase. And I agree with @snoyer we should use argv[0] if possible for the help command.

Reply:
definitely make sense. Lets just switch AppName to an all capitalized. The change needed is actually the project name in CMakeLists.txt. I hope there is no other far reaching change because of this that I may be not seeing.

Reply:
> definitely make sense. Lets just switch AppName to an all capitalized. The change needed is actually the project name in CMakeLists.txt. I hope there is no other far reaching change because of this that I may be not seeing.

Do I do that (and the `argv` change too?) in this PR or do you want to double check it independently?

Reply:
go for it :)